ga4gh/avrotools.py

```diff
@@ -14,6 +14,68 @@
 LONG_MAX_VALUE = (1 << 63) - 1
 
 
+class CreationToolException(Exception):
+    """
+    Something went wrong in the CreationTool
+    """
+
+
+class CreationTool(object):
+    """
+    Provides methods for creating and populating objects with data
+    """
```

Comment:
Adding a `SchemaTools` class (it doesn't just generate based on schemas, it checks against them, etc.) is a huge refactor.  I'll submit that in another branch.